deploy/helm/kubecf/templates/uaa_autoscaler_client.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.features.autoscaler.enabled -}}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}.var-cf-client-id
+  namespace: {{ .Release.Namespace | quote }}
+type: Opaque
+stringData:
+  value: app_autoscaler
```

Comment:
@f0rmiga This part seems weird… shouldn't that just be adding to `/variables/`? What am I missing?

Reply:
Yeah, I missed this in my first review.

Reply:
Removed this file and add it to the main autoscaler ops file.